logrotate.c

```diff
@@ -752,7 +752,7 @@ static int compressLogFile(const char *name, const struct logInfo *log, const st
     char *prevCtx;
     pid_t pid;
 
-    message(MESS_DEBUG, "compressing log with: %s\n", log->compress_prog);
+    message(MESS_DEBUG, "compressing log with %s: %s\n", log->compress_options_count ? "added options" : "default", log->compress_prog);
```

Comment:
The updated message sounds confusing to me:

> compressing log with default: /bin/gzip

I would rather understand it as if /bin/gzip was the default compress program.

What is the motivation behind this change?

If we want to increase the verbosity, we should rather print the array passed to `execvp()`.  But is it really needed? 

The original string that the array is build from is already printed while parsing the configuration...

Reply:
Yeah, you're right; this is not needed.